roles/foreman_testing/tasks/main.yaml

```diff
@@ -0,0 +1,10 @@
+---
+- name: "Run bats tests"
+  include_role:
```

Comment:
I think you can also define `meta/main.yml` with
```
dependencies:
  - { role: 'bats', when: foreman_testing_bats }
  - { role: 'hammer_tests', when: foreman_testing_hammer_tests }
```

Not sure what is best, tho. (https://docs.ansible.com/ansible/latest/playbooks_reuse_roles.html says "You can conditionally execute a role. This is not generally recommended with the classic syntax, but is common when using `import_role` or `include_role`")